src/vs/workbench/contrib/webview/common/resourceLoader.ts

```diff
@@ -94,7 +95,18 @@ function normalizeRequestPath(requestUri: URI) {
 	return requestUri.with({ scheme: 'file' });
 }
 
-function containsResource(root: URI, resource: URI): boolean {
-	const rootPath = root.fsPath + (endsWith(root.fsPath, sep) ? '' : sep);
+function containsResource(root: URI, resource: URI, extensionLocation: URI | undefined): boolean {
+	let rootPath = root.fsPath + (endsWith(root.fsPath, sep) ? '' : sep);
+	let resourceFsPath = resource.fsPath;
+
+	// windows paths are case insensitive
+	// when checking remote host, the host name is uppercase
+	if (extensionLocation && (
```

Comment:
Are you sure this is the correct fix? This code only applies if the extension is running remotely (which seems unrelated to #83646)

Reply:
`isUNC(root.fsPath) && isUNC(resource.fsPath)` this should do.